x-pack/plugins/ml/common/types/ml_url_generator.ts

```diff
@@ -8,6 +8,8 @@ import { RefreshInterval, TimeRange } from '../../../../../src/plugins/data/comm
 import { JobId } from './anomaly_detection_jobs/job';
 import { ML_PAGES } from '../constants/ml_url_generator';
 import { DataFrameAnalysisConfigType } from './data_frame_analytics';
+import { SearchQueryLanguage } from '../constants/search';
```

Comment:
Nit: Could be `import type ...`, could help keeping the page load bundle size down since I think we expose the url generator.